### PR TITLE
Support configurable CloudWatch alarm thresholds

### PR DESCRIPTION
**Pull request by nirojshrestha019**
Issue #, if available:

Description of changes:

Checklist:

* [ ] Have you successfully deployed to an AWS account with your changes?
* [ ] Have you written new tests for your core changes, as applicable?

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -187,6 +187,11 @@ oAuth2ApiEndpoint="undefined"
 patientPickerEndpoint="undefined"
 stage="dev"
 region="us-west-2"
+lambda_latency_threshold=3000
```
Comment (jmrah) [inline scripts/install.sh:190]: I was going to suggest you be consistent with your casing, perhaps following camelCaselike the other options, but then I inspected the script further, the casing convention is all over the place anyway.  Nice.  Carry on :)

File: scripts/install.sh [hunk 2]
```diff
@@ -206,6 +211,21 @@ while [ "$1" != "" ]; do
         -r | --region )                 shift
                                         region=$1
                                         ;;
+        -llt | --lambda_latency_threshold )    shift
```
Comment (jmrah) [inline scripts/install.sh:214]: Single dash options like you are using here are by far and away only used with single characters.  When they are used with more characters (such as the unix CLI command `tar -xvf` for example), each character stands for an individual option.  So, I would suggest just getting rid of the single dash options altogether in this case, and just keeping the double-dash longer ones.

**Review by lquanx — APPROVED** [review]

File: scripts/install.sh [hunk 3]
```diff
@@ -206,6 +211,21 @@ while [ "$1" != "" ]; do
...
+                                        lambda_latency_threshold=$1
+                                        ;;
+        -a5et | --apigateway5xxerror_threshold )    shift
```
Comment (jmrah) [inline scripts/install.sh:217]: You might want to update the usage() function in this script with these new options.

File: scripts/install.sh [hunk 4]
```diff
@@ -187,6 +187,11 @@ oAuth2ApiEndpoint="undefined"
 patientPickerEndpoint="undefined"
 stage="dev"
 region="us-west-2"
+lambda_latency_threshold=3000
+apigateway5xxerror_threshold=3
+apigateway4xxerror_threshold=5
+lambda_error_threshold=1
+DDBToESL_lambda_error_threshold=1
```
Comment (jmrah) [inline scripts/install.sh:194]: Are you sure the abbreviation is ESL?  Should it just be ES?

**Review by jmrah — CHANGES_REQUESTED** [review]

**Pull request closed without merging**